chain/jsonrpc/src/lib.rs

```diff
@@ -70,6 +74,10 @@ pub struct RpcConfig {
     pub polling_config: RpcPollingConfig,
     #[serde(default)]
     pub limits_config: RpcLimitsConfig,
+    // If true, enable some debug RPC endpoints (like one to get the latest block).
+    // We disable it by default, as some of those endpoints might be quite CPU heavy.
+    #[serde(default = "default_enable_debug_rpc")]
+    pub enable_debug_rpc: bool,
```

Comment:
I’m tempted to say we should just always enable it without configuration option. This doesn’t leak any sensitive information (I don’t think) and if someone has access to JSON RPC end point they can easily issue more expensive requests if they want to DoS the node.

Reply:
I'd rather play it safe - and start with disabled. We can always enable it in the future with 1-line PR

Reply:
Second mina86's opinion. I do think that we can enable this later, but this would be more risky: it's easier to audit new code for production readiness than to audit existing debug-only code.